snuba/clickhouse/optimize/optimize_scheduler.py

```diff
@@ -79,66 +79,50 @@ def subdivide_partitions(
         replacements.
         """
 
-        sorted_partitions = OptimizeScheduler.sort_partitions(partitions)
+        sorted_partitions = OptimizeScheduler._sort_partitions(partitions)
         output: MutableSequence[Sequence[str]] = []
 
         for i in range(number_of_subdivisions):
             output.append(sorted_partitions[i::number_of_subdivisions])
 
         return output
 
-    def start_time_jitter(self) -> Sequence[int]:
```

Comment:
Was this removed because this method is unused?

Reply:
yep: https://github.com/getsentry/snuba/pull/6175#discussion_r1735378797